pkgs/applications/editors/vscode/default.nix

```diff
@@ -84,14 +85,8 @@ in
         --set-rpath "${rpath}" \
         $out/lib/vscode/${executableName}
 
-      patchelf \
-        --set-rpath "${rpath}" \
-        $out/lib/vscode/resources/app/node_modules.asar.unpacked/keytar/build/Release/keytar.node
-
-      patchelf \
-        --set-rpath "${rpath}" \
-        "$out/lib/vscode/resources/app/node_modules.asar.unpacked/native-keymap/build/Release/\
-      keymapping.node"
+      find $out/lib/vscode/resources/app/node_modules.asar.unpacked -type f -name "*.node" \
```

Comment:
Should I expand this to every module instead of using find? Most of the native node modules need to be patched with `libstdc++.so.6`.

Reply:
You can have a look at `autoPatchelfHook`. This should patch most files automatically.